packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -45,6 +45,7 @@ BuildRequires: rubygem(apipie-rails) >= 0.5.14
 BuildRequires: rubygem(fx) < 1.0
 BuildRequires: rubygem(pg)
 BuildRequires: rubygem(anemone)
+BuildRequires: rubygem(spidr)
```

Comment:
You shouldn't add this in the same commit because it breaks things. It's a limitation in our build system, but you can only do this one this PR has been merged.